auto-cpufreq-installer

```diff
@@ -125,12 +126,21 @@ function tool_install {
 
   header "Installing necessary Python packages"
 
+<<<<<<< HEAD
```

Comment:
Are you sure you want to merge this like this, it seems like you were in process of resolving a git commit?